lib/iden3comm/domain/entities/common/iden3_message_entity.dart

```diff
@@ -3,20 +3,21 @@ enum Iden3MessageType {
   authRequest,
   authResponse,
   credentialOffer,
+  onchainCredentialOffer,
   credentialIssuanceResponse,
   proofContractInvokeRequest,
   credentialRefresh,
-  credentialStatusUpdate,
+  credentialStatusUpdate;
```

Comment:
typo?

Reply:
Nah, just different way of postpending last one
After ; you can add methods, fields, etc